guide/like.md

```diff
@@ -0,0 +1,10 @@
+## User Handbook
+### Liking a post
```

Comment:
For things like these, we should probably mention that an extension needs to be enabled for this feature to be available. To prevent confusion...